system/jlib/jlz4.hpp

```diff
@@ -22,7 +22,7 @@
 
 #define LZ4COMPRESSEDFILEBLOCKSIZE (0x100000)
 
-extern jlib_decl ICompressor *createLZ4Compressor(bool hc=false);
+extern jlib_decl ICompressor *createLZ4Compressor(const char * options, bool hc=false);
```

Comment:
I might have put options as the last arg and made it default to nullptr but nbd.